README.md

```diff
@@ -114,6 +114,40 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+#### Publishing messages with schema
+
+Ably Kafka connector supports messages with schema by converting schematic messages to JSON. Produced JSON is then passed as data to Ably message. JSON is produced by eliminating schema fields from the original message.
+For example if you use `AvroConverter` and register an Avro schema to your schema registry, you can use the following command providing your own schema and data:
+
+```shell
+ docker-compose exec -T schema-registry kafka-avro-console-producer \
+   --topic topic1 \
+   --broker-list kafka:9092 \
+   --property key.schema='{"type":"string"}' \
+   --property parse.key=true \
+   --property key.separator=":" \
+   --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"count","type":"int"}]}' \
+   --property schema.registry.url=http://schema-registry:8081 <<EOF
+"key1":{"count":1}
+"key2":{"count":2}
+"key3":{"count":3}
+EOF
+```
+
+You should receive following JSON messages where you subscribed:
+
+```json
+{"count":1}
```

Comment:
The user will also get the Avro-formatted key base64-encoded in the extras, for example:

```json
{
  "id": "1543960661:0:5",
  "clientId": "kafka-connect-ably-example",
  "connectionId": "sY0XR-p0oW",
  "timestamp": 1653912887495,
  "encoding": "json",
  "extras": {
    "kafka": {
      "key": "AAAAAAEIa2V5Mw=="
    }
  },
  "data": "{\"count\":3}"
}
```

I think it's useful to demonstrate that here.

Reply:
Added an example that contains whole message d5edbd101b440c36017a5ebf16ebcc2f95cbe24c